lib/barcodes_page.dart

```diff
@@ -27,12 +31,32 @@ class BarcodesPage extends StatelessWidget {
               child: Row(
                 children: [
                   Expanded(
-                    child: BarcodeItemWidget(barcode: leftBarcode),
+                    child: GestureDetector(
```

Comment:
Ten kod, który tworzy child dla lewego i prawego kodu wygląda identycznie. Trzeba to wynieść do oddzielnej funkcji która przyjmie w parametrach context i barcode